src/api/controllers/patient/patient.controller.ts

```diff
@@ -224,6 +224,12 @@ export class PatientController extends BaseUserController {
             if (!deleted) {
                 throw new ApiError(400, 'User cannot be deleted.');
             }
+            // invalidate all sessions
+            var invalidated = await this._userService.invalidateSession(request.currentUser.SessionId);
```

Comment:
Why you are sending session Id for this?
It should invalidate all active sessions for that user.

Reply:
Create a separate method which expires all the active sessions.
This should be separate from the method which expires a single session during logout.

Reply:
Understood. I will update this